src/main/java/net/dv8tion/jda/api/events/GenericEvent.java

```diff
@@ -37,4 +38,28 @@
      * @return The current sequence number for this event
      */
     long getResponseNumber();
+
+    /**
+     * The passthrough data that this event was serialized from.
+     * <br>This provides the full gateway message payload, including sequence and dispatch type.
+     * For details, read the official <a href="https://discord.dev/topics/gateway" target="_blank">Discord Documentation</a>.
+     *
+     * @throws IllegalStateException 
+     *         If the event has no raw data, see {@link #hasRawData()}
+     *
+     * @return The corresponding {@link DataObject}
+     *
+     * @see    #hasRawData()
+     */
+    @Nonnull
+    DataObject getRawData();
```

Comment:
Lets update the docs link to this https://discord.com/developers/docs/topics/gateway#commands-and-events-gateway-events